extensions/wikia/AssetsManager/configGroups/VenusConfig.php

```diff
@@ -82,6 +82,7 @@
 		'//extensions/wikia/JSSnippets/js/JSSnippets.js',
```

Comment:
User login and signup on venus hasn't been set up yet, so the changes in this file are just some cosmetic ones until we actually work on the venus skin.